### ISSUE TITLE
Make accessibility features editable

### ISSUE THREAD

**Issue by joedolson** [joedolson/my-calendar#623]
**Is your feature request related to a problem? Please describe.**

Currently, the accessibility features are stored in a way that makes them very difficult to extend or modify. It would be beneficial to end users if they could edit the features to better meet their needs, by renaming, adding more, or even removing. 

**Describe the solution you'd like**
Users should be able to add or edit the available accessibility features

Accessibility features should be searchable & filterable in event views.

**Comment by joedolson** [comment]
- [ ] Add event taxonomy for accessibility features
- [ ] Auto-populate taxonomy with existing options.
- [ ] Add ability to input or edit options. (Clone of categories?)
- [ ] Migrate event data to taxonomies
- [ ] Fix search & filter on accessibility features

Repeat with locations

**Comment by joedolson** [comment]
Filtering works; now just need to implement action scheduler for updates on large sites

**Comment by joedolson** [comment]
Needs some further testing, but it's *basically* done.
